File: EjemploPOO.cpp
class Contacto
{
	//atributos de contacto
	public char nombre[30];
	public long telefono;

	//metodos de contacto
   	public Contacto(char n[30], long t){
	strcpy (this.nombre, n)
	this.telefono = t;
	}

	void escribir(){
	printf("Nombre: %s telefono: %ld", this.nombre, this.telefono);
	}

	void cambiar(){
	this.telefono = nt;
	}
	
}

main(){

	Contacto c1;

	c1 = new Contacto("Pepe",1414);
	c1.escribir();
	c1.cambiar();
}

File: ejemplopoo.js
class Contacto{

	//creacion del constructor
	constructor(n, t){
		this.nombre = n;
		this.telefono = t;

	}

	Escribir(){
		var resultado = document.getElementById('Resultado')
		resultado.innerHTML += "Nombre: " + this.nombre + "<br>" + "Telefono: " + this.telefono + "<br>";
	}

	Cambiar(nuevoT){
		this.telefono = nuevoT;

	}

}

function CrearContacto(){
	
	 var c1 = new Contacto("pepe",1111),
	     c2 = new Contacto("ana", 2222);

	c1.Escribir();
	c2.Cambiar(3333);
	c2.Escribir();
}
